src/System.Device.Gpio/System/Device/Gpio/Drivers/LibgpiodCapiDriver.cs

```diff
@@ -0,0 +1,275 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Runtime.InteropServices;
+
+namespace System.Device.Gpio.Drivers
+{
+    public class LibgpiodCapiDriver : GpioDriver
+    {
+        private const string BasePath = "/dev/";
+
+        public string GpioChip { get; set; } = "gpiochip0";
```

Comment:
why do we default this to chip0?

Reply:
because that is the default chip (port) name for Rasp PI at least, didn't tried with other devices